### ISSUE TITLE
Untamed Wilds incompatibility

### ISSUE THREAD

**Issue by feliday** [Luke100000/minecraft-comes-alive#737]
Playing version 7.4.8 1.18.2 on Forge. Over the duration of playing, it seems that traits within a village start spreading to all the other villagers. One villager will be diabetic, and over time *all* the villagers will be diabetic. Or have heterochromia, be vegetarian, be bisexual, etc. This has been happening tenfold since opening an Inn, which brings in new traits. I have several other mods installed so it may be an issue related to that, but it's rather bothersome so I'd like to know!

**Comment by Luke100000** [comment]
No, there is no such thing implemented (except inheritance to offspring) and I doubt any mod could influence that. Please keep your eyes open for a way to reconstruct that. Tho I kinda like that idea, for e.g. diseases XD

**Comment by feliday** [comment]
I just figured it out after posting ahah.. Aside from MCA Reborn, the mods I have installed are mostly centred around animals. The mod "Untamed Wilds" has a crate which allows you to pick up entities, and I'd been using it to move villagers around. I believe the traits I had been picking up with the crate were all consolidating with each new villager I moved. Most of my village now being Diabetic, Vegetarian, Bisexual, Heterochromia. It was obvious in hindsight but without thinking it just looked like my village was being taken over by the disease known as being Vegetarian XD
